### PR TITLE
Add solution for Merge Two Binary Trees

### PR DESCRIPTION
**Pull request by Desgard**
The solution of the [Merge Two Binary Trees](https://leetcode.com/problems/merge-two-binary-trees/description/).

Signed-off-by: Desgard_Duan <gua@desgard.com>

### CODE REVIEW

File: Sort/ArrayPartitionI.swift [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Solution {
```
Comment (soapyigu) [inline Sort/ArrayPartitionI.swift]: Change class name to question title please

File: Tree/MergeTwoBinaryTrees.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ *     public var left: TreeNode?
+ *     public var right: TreeNode?
+ *     public init(_ val: Int) {
+ *         self.val = val
+ *         self.left = nil
+ *         self.right = nil
+ *     }
+ * }
+ */
+class Solution {
```
Comment (soapyigu) [inline Tree/MergeTwoBinaryTrees.swift]: Change class name to question title please

File: Sort/ArrayPartitionI.swift [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    func arrayPairSum(_ nums: [Int]) -> Int {
+        var arr = nums
+        arr = arr.sorted { $0 < $1 }
+        var res = 0
+        for i in 0 ..< arr.count {
+            if i & 1 != 0 {
+                res += min(arr[i], arr[i - 1])
+            }
+        }
+        return res
```
Comment (soapyigu) [inline Sort/ArrayPartitionI.swift]: All 10 lines could be replaced by 1 line: return nums.sorted(by: <).enumerated().flatMap { $0 % 2 == 0 ? $1 : nil }.reduce(0) { $0 + $1 }

Comment (Desgard) [inline Sort/ArrayPartitionI.swift]: Awesome! It's so cool! 👍

**Comment by Desgard** [comment]
And should I add the problem solving into the **Problem Status**? 
I found that the problem index is consequent. 😳

File: README.md [hunk 4]
```diff
@@ -767,3 +770,5 @@
 |	[Swift](./String/LongestSubstringWithoutRepeatingCharacters.swift)	|	3	|	[Longest Substring Without Repeating Characters](https://leetcode.com/problems/longest-substring-without-repeating-characters/)	| Medium	|
 |	[Swift](./Math/AddTwoNumbers.swift)	|	2	|	[Add Two Numbers](https://leetcode.com/problems/add-two-numbers/)	| Medium	|
 |	[Swift](./Array/TwoSum.swift)	|	1	|	[Two Sum](https://leetcode.com/problems/two-sum/)	| Easy	|
+
```
Comment (soapyigu) [inline README.md]: why keep 2 blank lines here?

File: README.md [hunk 5]
```diff
@@ -188,6 +189,7 @@
 [Burst Ballons](https://leetcode.com/problems/burst-balloons/)| [Swift](./DP/BurstBalloons.swift)| Hard| O(n^3)| O(n)|
 [Frog Jump](https://leetcode.com/problems/frog-jump/)| [Swift](./DP/FrogJump.swift)| Hard| O(n^2)| O(n)|
 
+
```
Comment (soapyigu) [inline README.md]: Why add 1 more line here?

**Pull request merged**